react-frontend/src/components/SelectInput/SelectInput.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './SelectInput.css'
+
+const SelectInput = ({ label, value, onChange }) => {
+  return (
+    <div className="select-input">
+      <label>{label}</label>
+      <select value={value} onChange={onChange}>
+        <option value="" disabled>
+          Select a category
+        </option>
+        <option value="category1">Category 1</option>
```

Comment:
Why `Category 1` (and others) are hardcoded here?

Reply:
It is temporary solution. Later I am going to rewrite the code so theat categories will be stored in a database and fetched them to display as options.